scripts/generate_mkdocs.py

```diff
@@ -3,15 +3,11 @@
 import re
 import shutil
 import subprocess
-import sys
 from pathlib import Path
 from typing import NamedTuple
 
 import yaml
 
-if sys.version_info < (3, 9):
-    raise RuntimeError("You need at least python 3.9 to run this script")
-
 
```

Comment:
Especially here, it seems that they should not be automatically removed

Reply:
Ah, I think this is "correct", but can we add `requires-python = ">=3.9"` to the `pyproject.toml` file in this directory? Right now it's defaulting to Python 3.10.

Reply:
Gonna add and merge.